src/App.test.tsx

```diff
@@ -432,3 +433,451 @@ test("renders slip legend", () => {
   expect(svgImg).toBeInTheDocument();
   expect(svgImg).toHaveClass("nothome-envelope img-fluid");
 });
+
+test("test language policy countable empty", () => {
+  const policy = new LanguagePolicy(2, "e");
```

Comment:
For the sake of maintainability, can i suggest that you use an existing constant here? So that if there is a change in policy, you just need to change the constant instead of all places with the hardcoded value, `2`.

Reply:
The value 2 is not really meant to be constant across all the tests. It forms a boundary that must be tested against the nhcount.